### PR TITLE
Helm release NS change to azure-arc-release

### PR DESCRIPTION
**Pull request by sirireddy12**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repository and upgrade the version in the pull request but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
connectedk8s

File: src/connectedk8s/azext_connectedk8s/_utils.py [hunk 1]
```diff
@@ -308,6 +308,8 @@ def helm_install_release(chart_path, subscription_id, kubernetes_distro, kuberne
                         "--set", "systemDefaultValues.spnOnboarding=false",
                         "--set", "global.azureEnvironment={}".format(cloud_name),
                         "--set", "systemDefaultValues.clusterconnect-agent.enabled=true",
+                        "--namespace", "{}".format(consts.Release_Install_Namespace),
```
Comment (akashkeshari) [inline src/connectedk8s/azext_connectedk8s/_utils.py:311]: add comment over this function, not to be used for re-put scenarios

Comment (akashkeshari) [reply src/connectedk8s/azext_connectedk8s/_utils.py:311]: check auto update logic if it takes --release-namespace correctly

Comment (sirireddy12) [reply src/connectedk8s/azext_connectedk8s/_utils.py:311]: Verified this. Autoupdate is picking up release namespace and it was able to upgrade successfully

**Review by akashkeshari — APPROVED** [review]

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**